### PR TITLE
Rename "timeseries" -> "time series"

### PR DESCRIPTION
**Pull request by danielnaab**
Before making a pull request
----------------------------

- [x] Make sure all tests run
- [x] Update the changelog appropriately

WDFN-239
-----------
Rename all references of "timeseries"

After making a pull request
---------------------------
- [x] If appropriate, put the link to the PR in the JIRA ticket
- [x] Assign someone to review unless the change is trivial

### CODE REVIEW

File: assets/src/scripts/components/hydrograph/parameters.js [hunk 1]
```diff
@@ -121,33 +121,33 @@ export const addSparkLine = function(svgSelection, {seriesLineSegments, scales})
...
     }
 
     const columnHeaders = ['Parameter', 'Preview', '#'];
     const tableContainer = elem.append('div')
-        .attr('id', 'select-timeseries');
+        .attr('id', 'select-time-series');
     tableContainer.append('label')
-        .attr('id', 'select-timeseries-label')
+        .attr('id', 'select-time-series-label')
         .text('Select a timeseries');
```
Comment (bdgarner-usgs) [inline assets/src/scripts/components/hydrograph/parameters.js]: Missed this one

Comment (danielnaab) [reply assets/src/scripts/components/hydrograph/parameters.js]: Ha, probably one of the only ones in the frontend! 👍

**Comment by bdgarner-usgs** [comment]
Looks good, except missed the "Select a timeseries" text.  I tried inserting a comment in the diff area here.  Did that work, and was it helpful/meaningful for you?

File: assets/src/scripts/components/hydrograph/parameters.js (hunk 1 above)
Comment (bdgarner-usgs) [reply assets/src/scripts/components/hydrograph/parameters.js]: Yes, exactly! :)

File: assets/src/scripts/components/hydrograph/cursor.js [hunk 2]
```diff
@@ -79,7 +79,7 @@ export const getNearestTime = function(data, time) {
 /*
  * Returns a function that the time series data point nearest the tooltip focus time for the given timeseries
```
Comment (ayan-usgs) [inline assets/src/scripts/components/hydrograph/cursor.js]: Not super critical, but you seem to have missed a `timeseries` in this line.

**Pull request merged**